types/backbone/index.d.ts

```diff
@@ -138,7 +138,7 @@ declare namespace Backbone {
         constructor(attributes?: any, options?: any);
         initialize(attributes?: any, options?: any): void;
 
-        fetch(options?: ModelFetchOptions): JQueryXHR;
+        fetch(options?: ModelFetchOptions): JQuery.Promise<any>;
```

Comment:
If this method actually returns a `jqXHR` object, you can replace it with `JQuery.jqXHR`.

Also, could you add a test for this?